### PR TITLE
Dbw node update

### PR DESCRIPTION
**Pull request by buffos**
- Change how we calculate CTE. Since we lowered the rate AND changed the message queue to the sim, the behavior of the controller changed. It looks as we are a bit late always. So instead of calculating the CTE at x=0, I do at x=5.0. I checked at various positions and it seems this value to be best. Also, it is in "sync" with the 1/5th of the rate we now have. Also now the vehicle drives fine in lane at all speeds (which I tried)

- Changed PID coefficients
- Changed filter application. Now the filter is applied to the errors (although the filter is disabled for now)
- Created a speed controller, based on torque
- Removed the old throttle PID from the controller
- Replaced with the new speed controller in dbw_node
- the usage is SpeedController.control(target_v, current_v, realization_time) . Realization time is the time I want that change to happen.
- Commented the code that we used to stop the car if speed is zero. It works fine without it now, but I did not delete it, in case we have problems.

It looks very stable now. Start and stop at traffic light works

### CODE REVIEW

File: ros/src/twist_controller/speed_controller.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#-------------------------------------------------------------------------------
+# Author: Kostas Oreopoulos <kostas.oreopoulos@gmail.com>
+# Date:   28.08.17
```
Comment (asimonov) [inline ros/src/twist_controller/speed_controller.py:3]: why 28th august?

**Review by asimonov — APPROVED** [review]
looks good

File: ros/src/twist_controller/speed_controller.py (hunk 1 above)
Comment (buffos) [reply ros/src/twist_controller/speed_controller.py:3]: Copy paste :)

**Comment by asimonov** [comment]
oh, I thought you’ve had it all along but was keeping it until the right moment :)

**Pull request merged**